### PR TITLE
Add Docker compose scripts

### PR DESCRIPTION
**Pull request by akphi**
Add and cleanup the Docker compose scripts based on the conversation in https://github.com/finos/legend/pull/363

### CODE REVIEW

File: docs/installation/installation-guide.md [hunk 1]
```diff
@@ -6,28 +6,32 @@ sidebar_label: Installation guide
```
Comment (epsstan) [inline docs/installation/installation-guide.md:6]: General. Default Legend Studio config has duplicate Gitlab entries. It also disables creation of prototype projects.

File: installers/docker-compose/legend-with-remote-gitlab/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Docker Compose for Legend with remote instance of Gitlab
```
Comment (epsstan) [inline installers/docker-compose/legend-with-remote-gitlab/README.md:1]: Please restructure the instructions such that there is a linear flow of steps. In its current form, the user has to go back and forth between the Gitlab OAuth configuration steps and this is not clear.

Comment (akphi) [reply installers/docker-compose/legend-with-remote-gitlab/README.md:1]: Gotcha, I have updated the README files with more detailed steps.

File: docs/installation/installation-guide.md (hunk 1 above)
Comment (akphi) [reply docs/installation/installation-guide.md:6]: Done!

**Comment by akphi** [comment]
@epsstan 
SOMETHING=http://example.org

GITLAB_OAUTH_APPLICATION=$(curl --request POST "https://localhost:443/api/v4/applications" --insecure --silent --header "PRIVATE-TOKEN: $GITLAB_PRIVATE_ACCESS_TOKEN" --data "name=Legend%20Demo2&redirect_uri=$SOMETHING/callback%0A$SOMETHING/api/auth/callback%0A$SOMETHING/api/pac4j/login/callback%0A$SOMETHING/studio/log.in/callback%0A&scopes=openid%0Aprofile%0Aapi&confidential=false")

**Review by epsstan — APPROVED** [review]

**Pull request merged**